cmd/server.go

```diff
@@ -126,6 +128,14 @@ var stringFlags = map[string]stringFlag{
 		description:  "Hostname of your Github Enterprise installation. If using github.com, no need to set.",
 		defaultValue: DefaultGHHostname,
 	},
+	GHTeamWhitelistFlag: {
+		description: "Comma separated list of key-value pairs representing the GitHub teams and the operations that the members of a particular team are allowed to perform. " +
+			"The format is {team}:{command},{team}:{command}, ex. dev:plan,ops:apply,devops:*. " +
+			"This example means to give the users from the 'dev' GitHub team the permissions to execute the 'plan' command, give the 'ops' team the permissions to execute the 'apply' command, " +
+			"and allow the 'devops' team to perform any operation. If this argument is not provided, the default value (*:*) will be used and the default behavior will be to not check permissions " +
+			"and to allow users from any team to perform any operation.",
```

Comment:
At least where it is easy to do so, please limit line length to 120 characters so that the code can be read on GitHub without horizontal scrolling. Although even this suggested change requires horizontal scrolling as a comment, at least it does not require horizontal scrolling when included as code.
```
		description: "Comma separated list of key-value pairs representing the GitHub teams and the " +
			"operations that the members of a particular team are allowed to perform. " +
			"The format is {team}:{command},{team}:{command}, ex. dev:plan,ops:apply,devops:*. " +
			"This example means to give the users from the 'dev' GitHub team the permissions to execute " +
			"the 'plan' command, give the 'ops' team the permissions to execute the 'apply' command, " +
			"and allow the 'devops' team to perform any operation. If this argument is not provided, " +
			"the default value (*:*) will be used and the default behavior will be to not check permissions " +
			"and to allow users from any team to perform any operation.",
```

Also, please list the valid values of `command`.

Reply:
made the description line length shorter than 120.
but in the original repo, they have lines with more than 180 chars (so this will not help much in readability)

Reply:
I think we should optimize for maintainability with the upstream. Anything we can/should do to reduce conflicts. This means not changing anything cosmetic.